spec/logux/action_controller_spec.rb

```diff
@@ -22,6 +21,23 @@
     end
   end
 
+  describe '#send_back' do
+    subject(:send_back) { action_controller.send_back(back_action) }
```

Comment:
let's check the case passing a meta, to ensure a result  of meta composition with `merge`

Reply:
Done. I suggested that checking both cases would be redundant, so I changed existing spec without meta passing instead of adding another one with passed meta. Let me know if I need to check both of these cases.